### PR TITLE
dropping behat tests from PdoEventSourcing module

### PR DESCRIPTION
**Pull request by unixslayer**

### CODE REVIEW

**Review by dgafka — APPROVED** [review]
Amazing refactor. You've put a lot of heart into this, thanks :) 

---
It's worth to give a context to this PR:
Ecotone Lite tests based on phpunit are much more easy to maintain than behat ones.
The behat tests were introduced before Ecotone Lite testing support were introduced and all new tests will be written in phpunit instead.

File: packages/PdoEventSourcing/tests/Integration/PollingProjectionTest.php [hunk 1]
```diff
@@ -4,31 +4,149 @@
...
+            ['ticket_id' => '124', 'ticket_type' => 'alert'],
+        ], $ecotoneLite->sendQueryWithRouting('getInProgressTickets'));
+
+        $ecotoneLite->deleteProjection(InProgressTicketList::IN_PROGRESS_TICKET_PROJECTION);
+        $ecotoneLite->run(InProgressTicketList::IN_PROGRESS_TICKET_PROJECTION);
+
+        self::assertFalse($connection->createSchemaManager()->tablesExist('in_progress_tickets'));
+    }
+
+    public function test_building_multiple_polling_projection(): void
```
[the remark this answers is not in the document]
Comment (unixslayer) [reply packages/PdoEventSourcing/tests/Integration/PollingProjectionTest.php:140]: About that maintainers thing ... I noticed there is only one at the moment ;)

[the remark this answers is not in the document]
Comment (dgafka) [reply packages/PdoEventSourcing/tests/Integration/PollingProjectionTest.php:140]: I think we can consider this one resolve now :)

[the remark this answers is not in the document]
Comment (jlabedo) [reply packages/PdoEventSourcing/tests/Integration/PollingProjectionTest.php:140]: > About that maintainers thing ... I noticed there is only one at the moment ;)

Haha 😉

**Pull request merged**